networkit/cpp/centrality/LocalSquareClusteringCoefficient.cpp

```diff
@@ -0,0 +1,53 @@
+// no-networkit-format
```

Comment:
We prefer to have new files formatted according to our coding style. Could you remove this line and clang-format the file?